source/documentation/administration_guide/index.adoc

```diff
@@ -29,4 +29,4 @@ include::appe-System_Accounts.adoc[]
 [appendix]
 == Legal notice
 
```

Comment:
If you don't want to create a module, at the very least I would create a 'year' attribute so that no one has to hunt down and update all these occurrences.